tests/project/compiler/test_solidity.py

```diff
@@ -58,11 +58,11 @@ def msolc(monkeypatch):
 
 def test_set_solc_version():
     compiler.set_solc_version("0.5.7")
-    assert solcx.get_solc_version().truncate() == compiler.solidity.get_version()
-    assert solcx.get_solc_version().truncate() == Version("0.5.7")
+    assert solcx.get_solc_version(True) == compiler.solidity.get_version()
```

Comment:
no idea what the boolean is used of the `get_solc_version()` call but it looks like `True` is used in every instance in this PR. Shouldn't that be the default?

Reply:
It will be the default moving on, we had to make some compiler changes and to not break anything we added `True` to use the updated version. Once everything is changed and working, we can move to the new version entirely.

Reply:
For readability, suppose it could say `with_commit_hash=True`